Sprint-2/errors/0.js

```diff
@@ -4,6 +4,10 @@
 // interpret the error message and figure out why an error is occurring
 
 function capitalise(str) {
-  let str = `${str[0].toUpperCase()}${str.slice(1)}`;
+  str = `${str[0].toUpperCase()}${str.slice(1)}`;
```

Comment:
You could also just return the value directly without first storing the value in a variable.